extension/manifest.json

```diff
@@ -9,7 +9,8 @@
         "128": "icons/icon-128.png"
     },
     "background": {
-        "scripts": ["scripts/background.js"]
+        "scripts": ["scripts/background.js"],
+        "persistent": true
```

Comment:
Why do you need it to be persistent ?

Reply:
From what I understood, a non-persistent script will re-run everytime there's a new event, and it may re-trigger the loading of the previous room. But I'm not really sure about what persistent actually does and if we're gonna need it with real time sync.